frontend/app/services/MemberService.scala

```diff
@@ -305,7 +305,7 @@ class MemberService(identityService: IdentityService,
 
   override def getUsageCountWithinTerm(subscription: Subscription, unitOfMeasure: String): Future[Option[Int]] = {
     val features = subscription.features
-    val startDate = subscription.startDate.toDateTimeAtCurrentTime()
+    val startDate = subscription.startDate.toDateTimeAtStartOfDay(DateTimeZone.forID("America/Los_Angeles"))
```

Comment:
I spent my train journey home digging and it looks like startDate used to refer to the term start date. Finding this confusing I rather dangerously changed it shortly before I left: https://github.com/guardian/membership-frontend/commit/7216c6c35c9d96fe615e75276bf1d6a902679abd#diff-abb6c51f77945af0f3a1eca1252ea19bL298

So hopefully we at least got 8 months of working tickets between Feb and Oct, and term start date is definitely what it should be